maas/plugins/nova_cloud_stats.py

```diff
@@ -82,8 +82,13 @@ def check(auth_ref, args):
         stats = nova.hypervisor_stats.statistics()
         cloud_stats = collections.defaultdict(dict)
         for metric_name, vals in stats_mapping.iteritems():
+            multiplier = 1
+            if metric_name == 'total_vcpus':
+                multiplier = args.cpu_allocation_ratio
+            elif metric_name == 'total_memory':
```

Comment:
Because used_memory and free_memory are the actual real values, and need to be for this whole thing to work. Only the totals need to be artificially multiplied to match the allocation_ratios